src/api/writers.ts

```diff
@@ -0,0 +1,17 @@
+import { Writer } from './indexer/types';
+import { Alias } from '../../.checkpoint/models';
+
+export function createWriters(indexerName: string) {
+  const handleSetAlias: Writer = async ({ unit, payload }) => {
+    const [from, to, salt] = payload.data;
+
+    const alias = new Alias(salt, indexerName);
```

Comment:
I got `0x0` as id, is this normal?

<img width="522" alt="image" src="https://github.com/user-attachments/assets/3fd1b705-5e7b-4379-a4ee-f7b9bfeeb2cb" />

I think the id should be `${from}:${alias}` if we want to have ability to remove record later (for example for an alias revoke action).

Reply:
`0x0` is normal because that's the salt I used in the payload.

I started with `${from}:${alias}` initially, but it became an issue if someone would try to create the same alias twice (which would work at agent level, but at API level we would need to think on how to handle it, in that case do we update `created` timestamp only?)